packages/fast-usdc/src/exos/README.md

```diff
@@ -0,0 +1,54 @@
+##   **StatusManager** sequence diagram, showing `TxStatus` state transitions
+
+### Current Implementation
```

Comment:
yay for diagrams!

I find the sequence diagram confusing for describing state transitions. Please try a state diagram:
```mermaid
stateDiagram-v2
    [*] --> Observed: EventFeed .observe()
    Observed --> Advanced: Advancer .advance()
    Advanced --> Settled: Settler .settle() after fees
    Observed --> Settled: Settler .settle() sans fees
    Settled --> [*]
```

I think it'll help even more when considering adding states and transitions